ibis/backends/duckdb/__init__.py

```diff
@@ -988,6 +988,47 @@ def read_postgres(
 
         return self.table(table_name)
 
+    ########
+    def read_mysql(
+        self,
+        uri: str,
+        catalog: str,
```

Comment:
@gforsyth I don't know if this is the best choice of word. Talking to Phillip he mentioned that these aliases are registered as catalogs, hence the choice, but feel free to suggest something different.

Reply:
`catalog` seems like right thing here given the direction we're going with naming those sorts of things.